test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/FunctionAssemblyLoadContextEndToEndTests.cs

```diff
@@ -40,34 +40,6 @@ public async Task Fallback_IsThreadSafe()
             });
         }
 
-        [Fact]
-        public async Task NativeDependency_Quirks()
```

Comment:
This scenario is no longer applicable and we can't build the target for Functions 2.0